### DISCUSSION TITLE
Can blitzjs be used with ant design pro?

### DISCUSSION THREAD

**Discussion by kjdion84** [blitz-js/blitz#2967] [Q&A] [answered]
If so, how do I use ant design pro with blitzjs?

I see the ant design pro has their own routing and command to install it.

I'm just wondering if this is possible, if it is not, what is a good admin UI you recommend to use with blitzjs?

**Comment by beerose** [comment, marked as answer]
It looks like Ant PRO is a whole scaffolder for admin apps — it creates a new application for you with Ant library and bootstrapped code to build an admin panel. It's a custom framework, with its own solutions. 
Blitz also scaffolds a project for you, but it's built on top of Next, so follows all of its core principles.
Hence I think you can consider using the `ant` design library with Blitz (https://ant.design/docs/react/introduce) instead of creating a project with a `pro` version. It should work fine with Blitz. You can also create a project with `ant pro` and copy paste some code to your Blitz app (for example, some UI components).

**Comment by yf-hk** [comment]
It doesn't seem so. However it's possible that blitz.js can work with pro-layouts, which includes a lot of ant design pro features
